src/donejs-carousel-plugin.js

```diff
@@ -480,16 +485,12 @@ export const ViewModel = DefineMap.extend({
 	* @param {number} pointerPosition (in pixels)
 	*/
 	moveCarouselToPosition(pointerPosition) {
-		let classSelector = this.classSelector;
+		const element = document.querySelector(this.classSelector);
 
```

Comment:
Just curious, how come it was necessary to move everything from jquery to vanilla js?

Reply:
it is much easier to read though!

Reply:
not strictly necessary but there are a few factors:
1) i read somewhere that a future version of canjs will not include jQuery by default (wish i could find the page, i tried looking but couldn't find it)
2) i was getting an error including jQuery after upgrading dependencies
3) will make our package more lightweight and portable